src/frontend/context/ConfigContext.tsx

```diff
@@ -67,19 +67,23 @@ export type State = {
   fields: FieldsMap;
   metadata: Metadata;
   messages: Messages;
-  status: Status;
+  status?: Status;
+  practiceModeBar: boolean;
+  practiceModeBarToggle: React.Dispatch<React.SetStateAction<boolean>> | null;
```

Comment:
curious if the practice mode should be tied to settings instead of the config. guess it's derived from the config but not sure what that means in terms of where it should live

Reply:
realized this is specific to the bar at the bottom and not practice mode as a whole. see [this comment](https://github.com/digidem/mapeo-mobile/pull/724/files#r698557849) for suggestion